tests/cephfs/cephfs_mirroring/test_cephfs_mirror_disconnect.py

```diff
@@ -0,0 +1,461 @@
+            f"--files 5 --files-per-dir 1 --dirs-per-dir 2 --top "
+            f"{kernel_mounting_dir_1}/{subvol1_path}/{mounting_dir}_mds",
+        )
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"mkdir {kernel_mounting_dir_1}{subvol1_path}.snap/{snap1}_mds",
+        )
+        with parallel() as p:
+            for mds_node in mds_nodes:
+                p.spawn(fs_util_ceph1.network_disconnect_v1, mds_node, 60)
+
+        fs_mirroring_utils.validate_snapshot_sync_status(
+            cephfs_mirror_node,
+            source_fs,
+            f"{snap1}_mds",
+            fsid,
+            asok_files,
+            filesystem_id,
+            peer_uuid,
+        )
+        out, rc = fs_mirroring_utils.list_and_verify_remote_snapshots_and_data_checksum(
+            target_clients[0],
+            "client.admin",
+            subvol1_path,
+            source_clients[0],
+            kernel_mounting_dir_1,
+            target_fs,
+        )
+        if not out:
+            raise CommandFailed(f"{rc}")
+        log.info("=" * 50)
+        log.info("Sceneario 4 : Disconnect mds nodes on target cluster")
+        log.info("=" * 50)
+        mds_nodes_target = fs_util_ceph2.get_mds_nodes(target_clients[0], target_fs)
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"mkdir -p {fuse_mounting_dir_1}/{subvol2_path}/{mounting_dir};"
+            f"python3 /home/cephuser/smallfile/smallfile_cli.py "
+            f"--operation create --threads 1 --file-size 1 "
+            f"--files 5 --files-per-dir 1 --dirs-per-dir 2 --top "
+            f"{fuse_mounting_dir_1}/{subvol2_path}/{mounting_dir}",
+        )
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"mkdir {fuse_mounting_dir_1}{subvol2_path}.snap/{snap2}_mds",
+        )
+        with parallel() as p:
+            for mds_node in mds_nodes_target:
+                p.spawn(fs_util_ceph2.network_disconnect_v1, mds_node, 60)
+
+        fs_mirroring_utils.validate_snapshot_sync_status(
+            cephfs_mirror_node,
+            source_fs,
+            f"{snap2}_mds",
+            fsid,
+            asok_files,
+            filesystem_id,
+            peer_uuid_new,
+        )
+        peer_uuid = peer_uuid_new
+        out, rc = fs_mirroring_utils.list_and_verify_remote_snapshots_and_data_checksum(
+            target_clients[0],
+            "client.admin",
+            subvol2_path,
+            source_clients[0],
+            fuse_mounting_dir_1,
+            target_fs,
+        )
+        if not out:
+            raise CommandFailed(f"{rc}")
+
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Clean up the system")
+        log.info("Delete the snapshots")
+        snapshots_to_delete = [
+            f"{kernel_mounting_dir_1}{subvol1_path}.snap/snap_k1*",
```

Comment:
please check the name of the snapshots to delete. snap_k1_nw and snap_f1_nw are the ones created earlier in the test.